classes/meson.yaml

```diff
@@ -52,9 +56,14 @@ buildScript: |
         if [[ ! -e .meson-done ]] ; then
             declare -a MESON_OPTIONS=( "$1" "$PWD")
             if [[ -e $BOB_CWD/cross_file.txt ]] ; then
+
+                if [[ "${MESON_FILE:-undefined}" != "undefined" ]]; then
+                    cat ${BOB_TOOL_PATHS['target-toolchain']}/${MESON_FILE} >> ../$cross_native
+                fi
                 MESON_OPTIONS+=("--cross-file" "$BOB_CWD/cross_file.txt")
                 MESON_OPTIONS+=("-Ddefault_library=${BASEMENT_LIBS:-shared}")
             else
+                MESON_OPTIONS+=("--native-file" "$BOB_CWD/native_file.txt")
```

Comment:
Why is this needed for host builds?

Reply:
I don't think it's necessary but with this the same file as for cross compiling can be used to tell meson about the system. For me this makes thinks more clear and the class easier to understand..